### DISCUSSION TITLE
Chairman Pause

### DISCUSSION THREAD

**Discussion by tristanBugeja** [AntonyCorbett/OnlyT#521] [Q&A]
Hello, I'm looking for a feature similar to a program we previously used, which allowed the chairman to time his comments by displaying a time prompts for his comments. I've have not found this functionality on OnlyT. Is there a way to incorporate this feature please?

**Comment by AntonyCorbett** [comment]
@tristanBugeja thanks for your question. Please search for “chairman” in the issues for several discussions about this.
